src/recorder/jamcontroller.cpp

```diff
@@ -26,7 +26,8 @@
 
 using namespace recorder;
 
-CJamController::CJamController() :
+CJamController::CJamController( CServer* server ) :
```

Comment:
Tiny point: the naming would be either `pServer` (pointer to Server Class) or `pNServer` (if you want to avoid the initialiser having the same name as the member - not that it matters - for "new" pointer to ...).

Reply:
Changed to `pNServer` in all three relevant places.